package.json

```diff
@@ -41,6 +42,7 @@
     "electron-promise-ipc": "^2.2.4",
     "electron-updater": "^4.1.2",
     "fs-extra": "^7.0.0",
+    "ios-deploy": "^1.11.4",
```

Comment:
Revert. This is a transitive dependency, I believe.

Reply:
It doesn't work for me, if i revert it

Reply:
https://github.com/Jigsaw-Code/outline-client/pull/1081#discussion_r717636656

Reply:
What version of NPM should I be using? Wonder if this is causing it, and the issue [here](https://github.com/Jigsaw-Code/outline-server/pull/993#discussion_r718841008)

https://user-images.githubusercontent.com/3759828/135340314-1aaa6845-9978-4df1-a7af-78b4e4975221.mov

Reply:
V7 seems to have fixed the `github:` thing i was seeing, but it still won't build unless i explicitly add `ios-deploy`

Reply:
https://github.com/apache/cordova-ios/issues/420

Reply:
using [`optionalDependencies`](https://docs.npmjs.com/cli/v7/configuring-npm/package-json#optionaldependencies) seems to be the solution - the ci ignores it, but it's installed locally in case the dev has their phone plugged in